src/SprykerEco/Zed/Payone/Business/Payment/MethodSender/AbstractPayoneMethodSender.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment\MethodSender;
+
+use Orm\Zed\Payone\Persistence\SpyPaymentPayone;
+use Orm\Zed\Payone\Persistence\SpyPaymentPayoneApiLog;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\AbstractRequestContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\CaptureContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\DebitContainer;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\RefundContainer;
+use SprykerEco\Zed\Payone\Business\Payment\PaymentMapperReader;
+use SprykerEco\Zed\Payone\Persistence\PayoneQueryContainerInterface;
+
+abstract class AbstractPayoneMethodSender
```

Comment:
* Maybe better name will be `AbstractPayoneRequestSender` ?

Reply:
++